docs/source/infrastructure.rst

```diff
@@ -28,9 +28,7 @@ We have stage and production deployments of Balrog. Here's a quick summary:
 Support & Escalation
 --------------------
 
-RelEng is the first point of contact for issues. To contact them, follow `the standard RelEng escalation path <https://wiki.mozilla.org/ReleaseEngineering#Contacting_Release_Engineering>`_.
-
-If RelEng is unable to correct the issue, they may `escalate to CloudOps <https://mana.mozilla.org/wiki/display/SVCOPS/Contacting+Services+SRE>`_.
```

Comment:
Are we actually losing SRE support? Aren't they still valid for escalation?

Reply:
We are definitely not losing their support :).

I mostly removed this because I was thinking that it's same as what we do for everything else deployed through SRE. However, maybe it's more important to keep this here because this an externally facing system...